doc/Performing_IO_Operations_Using_S3Client.rst

```diff
@@ -8,7 +8,8 @@ You need to install the S3 client to perform the IO operations in the CORTX. The
 
 For more information on:
 
--  Sample code for doing S3 IO operations, see `CORTX Sample S3 Code <https://github.com/Seagate/cortx/tree/main/cortx-s3samplecode>`__.
--  To use CyberDuck for IO operations, see `CyberDuck for IO Operations <https://github.com/Seagate/cortx/blob/main/doc/CyberDuckIO.md>`__.
--  AWS CLI on the same OVA, see `AWS CLI on the CORTX OVA VM <https://github.com/Seagate/cortx/blob/main/doc/aws_cli_on_same_vm.md>`__.
--  AWS CLI on the new VM, see `AWS CLI on a different VM <https://github.com/Seagate/cortx/blob/main/doc/aws_on_separate_vm.md>`__.
+-  Sample code for doing S3 IO operations, see `CORTX Sample S3 Code </cortx-s3samplecode>`__.
+-  To use CyberDuck for IO operations, see `CyberDuck for IO Operations <CyberDuckIO.md>`__.
+-  AWS CLI on the same OVA, see `AWS CLI on the CORTX OVA VM <aws_cli_on_same_vm.md>`__.
+-  AWS CLI on the new VM, see `AWS CLI on a different VM <aws_on_separate_vm.md>`__.
```

Comment:
@nrady, looks like we have to remove the S3 IO performance for single or different VM as CORTX GUI feature been deprecated in K8s architecture so would be better if you refer https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-5/S3_IO_Operations.md and update all the relative link accordingly.

Also, same is updated in https://github.com/Seagate/cortx/blob/main/QUICK_START.md

Reply:
Hi @mukul-seagate11 
       I updated the new PI-5 for new OVA with K8s in S3 IO test option list. To remain variety, I prefer to keep old one as a option for old OVA. Please consider. Thanks